tests/distributions/test_von_mises.py

```diff
@@ -14,3 +14,19 @@ def test_log_prob_normalized(concentration):
     prob = VonMises(0.0, concentration).log_prob(grid).exp()
     norm = prob.mean().item() * 2 * math.pi
     assert abs(norm - 1) < 1e-3, norm
+
+
+@pytest.mark.parametrize('scale', [0.1, 0.5, 0.9, 1.0, 1.1, 2.0, 10.0])
+def test_von_mises_3d(scale):
+    concentration = torch.randn(3)
+    concentration = concentration * (scale / concentration.norm(2))
+
+    num_samples = 100000
+    samples = torch.randn(num_samples, 3)
+    samples = samples / samples.norm(2, dim=-1, keepdim=True)
+
+    d = VonMises3D(concentration, validate_args=True)
```

Comment:
If we do not explicitly turn this flag to `True`, is the distributions arg checking disabled? We should have it enabled by default, unless there has been some regression lately.

Reply:
Good point, I'll update this.